main.js

```diff
@@ -35,4 +35,165 @@ menuCon.addEventListener('click', () => {
   mobileNav.classList.remove('active');
   background.classList.remove('blur');
   backgroundNav.classList.remove('blur');
-});
\ No newline at end of file
+});
+
+const projectSection = [
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-3.png',
+    technologies: ['html', 'css', 'javascript'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-3.png',
+    technologies: ['html', 'css', 'javascript'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-4.png',
+    technologies: ['html', 'css', 'Bootstrap'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-5.png',
+    technologies: ['html', 'css', 'React'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+];
```

Comment:
- [ ] Kindly make sure all the details in this section conform to the Figma template, currently you have the same text for all the Projects